test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/HelpInfoTests.cs

```diff
@@ -27,6 +27,8 @@ .NET Testing Platform v*
 Usage {TestAssetFixture.NoExtensionAssetName}* [option providers] [extension option providers]
 Execute a .NET Test Application.
 Options:
+    --config
```

Comment:
I mean wondering if we should have `--config-file` instead. @MarcoRossignoli @nohwnd any opinion?

Reply:
--config-file is good so